### ISSUE TITLE
Is a YANG submodule a YANG artifact

### ISSUE THREAD

**Issue by rgwilton** [netmod-wg/yang-ver-dt#26]
E.g. in the YANG semver draft.  This potentially has an impact on the packages/versioning drafts that refer to them.

**Comment by xorrkaz** [comment]
We had said that submodules would inherit the parent module version.  So in this case, no, a submodule is not an artifact.  The artifact is the module and all of its submodules.

**Comment by rgwilton** [comment]
I agree.

At some point, I thought that I need to refer to YANG package, module, or submodule, so was wondering if I can refer to those as an artifact, but I think that would be a mistake.

**Comment by xorrkaz** [comment]
Should I add some text to semver to make that clearer?

**Comment by rgwilton** [comment]
Not sure, probably not required.  Probably this issue can just be closed.

The problem was that I wanted to use a grouping to cover the URI + checksum, which is used in 5 places (e.g. for packages, twice for modules, twice for submodules).  For the moment I've just not bothered with the grouping, but potentially could use a top level grouping + refine statements instead to update the description for package vs module vs submodule.

When I looked to create "unified" description text then it started to look a bit ugly anyway.

So, in summary, probably a non issue.

**Comment by rgwilton** [comment]
No change required.
